contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -421,27 +423,26 @@ var BaseWorkspaceView = BaseView.extend({
 	},
 	open_archive:function(){
 		var ArchiveView = require("edit_channel/archive/views");
-		var archive = new ArchiveView.ArchiveModalView({
-			model : new Models.ContentNodeModel(State.current_channel.get("trash_tree"))
-	 	});
+		State.Store.dispatch('usePrimaryModal', () => {
+			return new ArchiveView.ArchiveModalView({
+				model : new Models.ContentNodeModel(State.current_channel.get("trash_tree"))
+			 });	
+		})
 	},
 	move_content:function(move_collection, source){
 		var MoveView = require("edit_channel/move/views");
 		var list = this.get_selected(true);
 		var move_collection = new Models.ContentNodeCollection(_.pluck(list, 'model'));
-		$("#main-content-area").append("<div id='dialog'></div>");
-
 		var self = this;
-		var move = new MoveView.MoveModalView({
+		return new MoveView.MoveModalView({
 			collection: move_collection,
-			el: $("#dialog"),
-        onmove: function(target, moved, original_parents) {
-          if (source === "clipboard") {
-            self.track_event_for_nodes('Clipboard', 'Move items', moved);
-          }
-          self.handle_move(target, moved, original_parents);
-        },
-		    model: State.current_channel.get_root("main_tree")
+			onmove: function(target, moved, original_parents) {
```

Comment:
Can use an arrow function here also if you want to avoid `self = this` - the `this` is bound to the arrow function at time of function declaration, which in this case is inside the `move_content` method invocation. Only if you used object method notation would the `this` shift its scope to something else (exactly what depends on how Backbone defines properties on Views from options).